raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -31,15 +31,24 @@ contract TokenNetwork is Utils {
         115792089237316195423570985008687907853269984665640564039457584007913129639935
     );
 
-    // Bug bounty release deposit limit
-    uint256 public deposit_limit;
+    // Bug bounty release deposit limits
+    // The combined deposit of one channel is limited to 0,15 ETH.
```

Comment:
nitpick: you are using a `,` for decimal separator here and a `.` in the line below. Choose one and stick to it.